diff -urN OpenSpliceDDS-6.4.1-orig/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/include/dcps/C++/isocpp/make_files.py OpenSpliceDDS-6.4.1/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/include/dcps/C++/isocpp/make_files.py
--- OpenSpliceDDS-6.4.1-orig/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/include/dcps/C++/isocpp/make_files.py	2018-12-03 17:57:11.374514000 +0000
+++ OpenSpliceDDS-6.4.1/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/include/dcps/C++/isocpp/make_files.py	2018-12-03 17:59:24.984626000 +0000
@@ -45,7 +45,7 @@
     myfile.write("#define OSPL_" + mymacro + "_\n")
     myfile.write("\n")
     myfile.write("/**\n")
-    myfile.write(" * @file
+    myfile.write(" * @file\n")
     myfile.write(" */\n")
     myfile.write("\n")
     if not "detail" in new_filename:
@@ -70,4 +70,4 @@
     myfile.write("// End of implementation\n")
     myfile.write("\n")
     myfile.write("#endif /* OSPL_" + mymacro + "_ */\n")
-    myfile.close()
\ No newline at end of file
+    myfile.close()
diff -urN OpenSpliceDDS-6.4.1-orig/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/release.com OpenSpliceDDS-6.4.1/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/release.com
--- OpenSpliceDDS-6.4.1-orig/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/release.com	2018-12-03 17:57:11.314514000 +0000
+++ OpenSpliceDDS-6.4.1/OpenSpliceDDS/V6.4.1/HDE/x86_64.linux/release.com	2018-12-03 17:58:45.734871000 +0000
@@ -9,7 +9,7 @@
     SPLICE_JDK=jdk
     export SPLICE_JDK
 fi
-#### set in setup.env ### OSPL_HOME="@@INSTALLDIR@@/HDE/x86_64.linux"
+OSPL_HOME="/opt/OpenSpliceDDS/HDE/x86_64.linux"
 PATH=$OSPL_HOME/bin:$PATH
 LD_LIBRARY_PATH=$OSPL_HOME/lib${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH
 CPATH=$OSPL_HOME/include:$OSPL_HOME/include/sys:${CPATH:=}
